### PR TITLE
Feat/update locker

### LINKED ISSUE #124: Update locker

### PR DESCRIPTION
**Pull request by StarryFolf**

### CODE REVIEW

File: src/Application/Lockers/Commands/UpdateLocker.cs [hunk 1]
```diff
@@ -1,15 +1,83 @@
...
+        public async Task<LockerDto> Handle(Command request, CancellationToken cancellationToken)
+        {
+            var locker = await _context.Lockers.FirstOrDefaultAsync(
+                x => x.Id.Equals(request.LockerId), cancellationToken);
+
+            var duplicateLocker = await _context.Lockers.FirstOrDefaultAsync(
+                x => x.Name.Equals(request.Name), cancellationToken);
+            
+            if (locker is null)
+            {
```
Comment (ChienNQuang) [inline src/Application/Lockers/Commands/UpdateLocker.cs:57]: this check should be before duplicateLocker

Comment (StarryFolf) [reply src/Application/Lockers/Commands/UpdateLocker.cs:57]: what?

File: src/Application/Departments/Queries/GetAllDepartments.cs [hunk 2]
```diff
@@ -24,9 +24,11 @@ public QueryHandler(IApplicationDbContext context, IMapper mapper)
 
         public async Task<IEnumerable<DepartmentDto>> Handle(Query request, CancellationToken cancellationToken)
         {
-            var departments = await _context.Departments.ToListAsync(cancellationToken);
+            var departments = await _context.Departments
+                .Where(x => !x.Name.Equals("Admin"))
```
Comment (ChienNQuang) [inline src/Application/Departments/Queries/GetAllDepartments.cs]: why hide department admin?

**Review by ChienNQuang — APPROVED** [review]
ok

**Pull request merged**